src/core_modules/capture-core/reducers/descriptions/activePage.reducerDescription.js

```diff
@@ -10,10 +10,6 @@ import { actionTypes as viewEventDataEntryActionTypes } from '../../components/P
 import { eventWorkingListsActionTypes } from '../../components/Pages/MainPage/EventWorkingLists';
 
 export const activePageDesc = createReducerDescription({
-    [lockedSelectorActionTypes.ORG_UNIT_ID_SET]: state => ({
-        ...state,
-        isDataEntryLoading: true,
-    }),
```

Comment:
also this caused some weird behaviour and didnt see where we make use of it. There is good chance that I am simply unaware of the use-case so please take a look.